### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by SaadMahi**
Design to code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,300&family=Work+Sans:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <!--Css Style Linked-->
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img src="" alt="" class="logo" />
```
Comment (murtazshaikh) [inline index.html]: src is empty ?

**Review by murtazshaikh — APPROVED** [review]
Nice

File: index.html (hunk 1 above)
Comment (SaadMahi) [reply index.html]: src given!

**Comment by zaidansari42** [comment]
Saad, Have you created the hamburger Menu? which was the main part of this project. And please add the transition time to all the links when hovered, so that you'll get smooth fade in hover effect !

**Review by ManSOorcode — APPROVED** [review]

**Comment by SaadMahi** [comment]
good to merge!

**Pull request merged**